package.json

```diff
@@ -35,8 +35,9 @@
     "custom-logger": "0.3.1",
     "inquirer": "0.10.0",
     "isemail": "1.2.0",
+    "lodash.isempty": "4.4.0",
```

Comment:
This extra dependency can be omitted by using `return (input == null)` in `utils.isNullOrUndefined`. That's exactly what `isnil` does: https://github.com/lodash/lodash/blob/6cb3460fcefe66cb96e55b82c6febd2153c992cc/isNil.js#L19